src/NuGet.Core/NuGet.Packaging/Strings.resx

```diff
@@ -653,8 +650,10 @@ Valid from:</comment>
   <data name="SignError_TimestampSignatureValidationFailed" xml:space="preserve">
     <value>The timestamp signature validation failed.</value>
   </data>
-  <data name="SignError_TimestampUnsupportedSignatureAlgorithm" xml:space="preserve">
-    <value>The timestamp certificate has an unsupported signature algorithm.</value>
+  <data name="SignError_TimestampCertificateUnsupportedSignatureAlgorithm" xml:space="preserve">
+    <value>The timestamp certificate has an unsupported signature algorithm - '{0}'. The following algorithms are supported - '{1}'.</value>
```

Comment:
Don't use a hyphen like this, and you don't need single quotation marks.  Instead:

    The timestamp certificate has an unsupported signature algorithm ({0}). The following algorithms are supported:  {1}.